### DISCUSSION TITLE
Electrolytic Capacitors

### DISCUSSION THREAD

**Discussion by AdamRoul** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#718] [Q&A]
Hello

Just building my board. I found that I already have some 220uf electrolytic capacitors.  Can I use these in the project?  I don't know a great deal about electronics but it seems that electrolytic capacitors have polarity - which way should I orientate them for this project?  Failing that, does anyone have a link to where I can source the right capacitors (for postage to the UK)?  Thanks

**Comment by visualapproach** [comment]
Try the A ports first. If it's not working you can buy caps+res to the B ports

**Reply by AdamRoul** [reply]
Hello.  Thanks for your reply.  I haven't been able to proceed from my earlier message because I found that I had ordered the wrong connections.  New connectors have arrived and I've set everything up.  All seems to be working (although I can't fully test till I actually fill the tub!), although the display isn't working. I assume this means I need the resistors and capacitors?  If so, can anyone help with my original question please?  As another question - the blue light on the NodeMCU is almost always flashing - is that ok?

**Reply by visualapproach** [reply]
Electrolytic caps are not usually used for this kind of application. So try non polarized, like ceramic.
You can read page 26 in the manual about the blue LED.
